src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkArtistMarket.tsx

```diff
@@ -112,7 +112,7 @@ const InsightColumn = ({ name, value }: { name: string; value: string }) => {
   return (
     <Column span={[6, 4, 2]}>
       <Flex flexDirection={"column"} justifyContent="flex-start">
-        <Text variant={["xs", "md", "md"]}>{name}</Text>
+        <Text variant={["xs", "sm-display", "sm-display"]}>{name}</Text>
         <Text variant={["lg", "xl", "xl"]}>{value}</Text>
```

Comment:
**Suggestion:** 
```suggestion
        <Text variant={["xs", "sm-display"]}>{name}</Text>
        <Text variant={["lg", "xl"]}>{value}</Text>
```

Reply:
ah oh cool, thanks, i thought there was a reason to keep all three

Reply:
There is a reason to have them sometimes, here's a guide about them in [Palette] :)(https://palette.artsy.net/guides/responsive/)

Reply:
Got it :)